docs/content/ChoiceGroup.mdx

```diff
@@ -0,0 +1,382 @@
+      setSelectedRadioValue(value)
+    }
+  }
+
+  return (
+    <Box display="grid" sx={{gap: 3}}>
+      <Box display="grid" sx={{gap: 1}}>
+        <ChoiceGroup>
+          <ChoiceGroup.Label>Checkboxes</ChoiceGroup.Label>
+          <FormControl>
+            <Checkbox value="checkOne" onChange={handleCheckboxChange} />
+            <FormControl.Label>Checkbox one</FormControl.Label>
+          </FormControl>
+          <FormControl>
+            <Checkbox value="checkTwo" onChange={handleCheckboxChange} />
+            <FormControl.Label>Checkbox two</FormControl.Label>
+          </FormControl>
+          <FormControl>
+            <Checkbox value="checkThree" onChange={handleCheckboxChange} />
+            <FormControl.Label>Checkbox three</FormControl.Label>
+          </FormControl>
+        </ChoiceGroup>
+
+        {Boolean(selectedCheckboxValues.length) && (
+          <div>The selected checkbox values are {selectedCheckboxValues.join(', ')}</div>
+        )}
+      </Box>
+
+      <Box display="grid" sx={{gap: 1}}>
+        <ChoiceGroup>
+          <ChoiceGroup.Label>Radios</ChoiceGroup.Label>
+          <FormControl>
+            <Radio name="radioChoices" value="radioOne" onChange={handleRadioChange} />
+            <FormControl.Label>Radio one</FormControl.Label>
+          </FormControl>
+          <FormControl>
+            <Radio name="radioChoices" value="radioTwo" onChange={handleRadioChange} />
+            <FormControl.Label>Radio two</FormControl.Label>
+          </FormControl>
+          <FormControl>
+            <Radio name="radioChoices" value="radioThree" onChange={handleRadioChange} />
+            <FormControl.Label>Radio three</FormControl.Label>
+          </FormControl>
+        </ChoiceGroup>
+
+        {selectedRadioValue && <div>The selected radio value is {selectedRadioValue}</div>}
+      </Box>
+    </Box>
+  )
+}
+
+render(<WithOnChangeHandlers />)
+```
+
+### Checkbox group
+
+```jsx live
+<ChoiceGroup>
+  <ChoiceGroup.Label>Choices</ChoiceGroup.Label>
+  <FormControl>
+    <Checkbox />
+    <FormControl.Label>Choice one</FormControl.Label>
+  </FormControl>
+  <FormControl>
+    <Checkbox />
+    <FormControl.Label>Choice two</FormControl.Label>
+  </FormControl>
+  <FormControl>
+    <Checkbox />
+    <FormControl.Label>Choice three</FormControl.Label>
+  </FormControl>
+</ChoiceGroup>
+```
+
+### Disabled
+
+```jsx live
+<ChoiceGroup disabled>
+  <ChoiceGroup.Label>Choices</ChoiceGroup.Label>
+  <FormControl>
+    <Radio name="radioInput" value="choiceOne" />
+    <FormControl.Label>Choice one</FormControl.Label>
+  </FormControl>
+  <FormControl>
+    <Radio name="radioInput" value="choiceTwo" />
+    <FormControl.Label>Choice two</FormControl.Label>
+  </FormControl>
+  <FormControl>
+    <Radio name="radioInput" value="choiceThree" />
+    <FormControl.Label>Choice three</FormControl.Label>
+  </FormControl>
+</ChoiceGroup>
+```
+
+### Required
+
+```jsx live
+<ChoiceGroup required>
```

Comment:
Would we ever want to allow the colour of the asterisk to be changed? For example, on some forms you'll see it in red for extra prominence.

Reply:
As of now, we do not. The forms as they exist today are totally custom and not following any kind of system.